stackit/internal/services/loadbalancer/loadbalancer/resource.go

```diff
@@ -55,10 +55,16 @@ type Model struct {
 
 // Struct corresponding to each Model.Listener
 type Listener struct {
-	DisplayName types.String `tfsdk:"display_name"`
-	Port        types.Int64  `tfsdk:"port"`
-	Protocol    types.String `tfsdk:"protocol"`
-	TargetPool  types.String `tfsdk:"target_pool"`
+	DisplayName          types.String          `tfsdk:"display_name"`
+	Port                 types.Int64           `tfsdk:"port"`
+	Protocol             types.String          `tfsdk:"protocol"`
+	ServerNameIndicators []ServerNameIndicator `tfsdk:"serverNameIndicators"`
```

Comment:
We have a known bug that using a custom type for the resource fields causes an error when providing the values as a variable. So we will replace all of these custom types by `types.Object`. 

We planned on doing it in this iteration, so we can jump in here and replace it for this field also

Reply:
Modified it